lib/webauto/web4cucumber.rb

```diff
@@ -137,20 +137,19 @@ def browser
       elsif @browser_type == :chrome
         logger.info "Launching Chrome"
 
-	      #https://bugs.chromium.org/p/chromium/issues/detail?id=1056073
-	      chrome_caps[:acceptInsecureCerts] = true
-        if Integer === @scroll_strategy
-          chrome_caps[:element_scroll_behavior] = @scroll_strategy
-        end
         if self.class.container?
           chrome_switches.concat %w[--no-sandbox --disable-setuid-sandbox --disable-gpu --disable-infobars --disable-dev-shm-usage]
         end
-        # options = Selenium::WebDriver::Chrome::Options.new
-        # options.add_extension proxy_chrome_ext_file if proxy_chrome_ext_file
-        options = {}
+        options = {browser_name: 'chrome',
+                   accept_insecure_certs: true}
+        options[:element_scroll_behavior] = @scroll_strategy if Integer === @scroll_strategy
         options[:extensions] = [proxy_chrome_ext_file] if proxy_chrome_ext_file
-        url_or_switches = @selenium_url ? {url: @selenium_url} : {switches: chrome_switches}
-        @browser = Watir::Browser.new :chrome, desired_capabilities: chrome_caps, options: options, **url_or_switches
+        if @selenium_url
+          @browser = Watir::Browser.new :chrome, options: options, url: @selenium_url
+        else
+          options[:switches] = chrome_switches
```

Comment:
Hi @jhou1  we need one more update here, we can't pass `switches` directly to `options`, instead we need put it into `options["goog:chromeOptions"]`
```
options["goog:chromeOptions"][:switches] = chrome_switches
```

Reply:
There are still some issues when testing against Proxy, debugging